plugin/index.tpl.html

```diff
@@ -73,6 +72,12 @@ <h1>How to use this plugin</h1>
 		<p>Alternatively, you can download the source files for this plugin by visiting its repository on Github</p>
 
 		<p><a href="https://github.com/[plugin.repo]" class="call-to-action button">Visit [plugin.repo]</a></p>
+
+		<div mv-if="plugin.url">
+			<p>Or you can download the JavaScript file of this plugin by clicking below:</p>
+
+			<p><a href="[plugin.url]" class="call-to-action button">Download mavo-[plugin.id].js</a></p>
```

Comment:
See comment above about the URL potentially not including the filename.
Also, I think having two links for downloading files is confusing.

Reply:
Yes, I think we don't need the second link here since we can download the files from the repo if needed.

Reply:
Also, if the plugin includes other resources, downloading just the JS file will break it.